MaterialDesignThemes.Wpf/ISnackbarMessageQueue.cs

```diff
@@ -51,7 +51,21 @@ public interface ISnackbarMessageQueue
         /// <param name="actionContent">Content for the action button.</param>
         /// <param name="actionHandler">Call back to be executed if user clicks the action button.</param>
         /// <param name="actionArgument">Argument to pass to <paramref name="actionHandler"/>.</param>
-        /// <param name="promote">The message will promoted to the front ot the queue and never considered to be a duplicate.</param>
-        void Enqueue<TArgument>(object content, object actionContent, Action<TArgument> actionHandler, TArgument actionArgument, bool promote);
```

Comment:
Hi @Keboo ...for the sake of backward compatibility, can we keep this method, but call your new method setting neverConsiderToBeDuplicate to equal promote.  That will also be in keeping with the existing method docs.

Reply:
Yes, and done.